pypeit/flatfield.py

```diff
@@ -137,8 +137,11 @@ def _bundle(self):
         # Return
         return d
 
+    # TODO: Although I don't like doing it, kwargs is here to catch the
+    # extraneous keywords that can be passed to _parse from the base class but
+    # won't be used.
     @classmethod
-    def _parse(cls, hdu, ext=None, transpose_table_arrays=False, hdu_prefix=None):
+    def _parse(cls, hdu, ext=None, transpose_table_arrays=False, hdu_prefix=None, **kwargs):
```

Comment:
I have previously named it `extras` instead of `kwargs` 
Maybe do the same here

Reply:
I'm fine with changing from `kwargs` to `extras`, but I think `kwargs` is the broadly accepted variable name.  Here's a random example: https://numpy.org/devdocs/reference/generated/numpy.ufunc.__call__.html#numpy.ufunc.__call__

Reply:
Any consensus on this?  I'm planning on leaving these instances as `kwargs` unless there are strong opinions otherwise.